src/main/play-doc/operation/Install.md

```diff
@@ -1399,11 +1193,11 @@ The bundle can now be scaled to run multiple instances
 $ conduct run --scale=3 visual
 ```
 
-Test service endpoints from the public nodes to ensure connectivity. The `haproxy.cfg` file, `/etc/haproxy/reloadHAProxy.sh` in the above, contains the frontend and backend configurations for services exposed by bundles. Test the front end bind port and path using `curl` and the public node ip address to ensure proxing is working correctly.  
+Test service endpoints from the public nodes to ensure connectivity. The `/etc/haproxy/haproxy.cfg` file on the public slaves contains the frontend and backend configurations for services exposed by bundles. Test the front end bind port and path using `curl` and the public node ip address to ensure proxing is working correctly.  
 
 
 ## Configuring the service gateway
 
-Now that we have configured ConductR-HAProxy to run on the public nodes, we'll want to configure them as a public service. If the public nodes have public IP addresses, the addresses can be listed in DNS directly. For better resiliency, use multiple public proxy nodes. 
+Now that we have configured ConductR-HAProxy to run on the public nodes, we'll want to configure them as a public service. If the public nodes have public IP addresses, the addresses can be listed in DNS directly. For better resiliency, use multiple public proxy nodes.
 
-If we do not want to have multiple "A" records, we can use a load balancer, such as EC2's ELB, for the service CNAME address. In doing such, we will create a new a load balancer instance for the ConductR services as this allows the load balancer single health check to use ConductR-HAProxy's health endpoint, not marathon-lb's.  In the example of AWS ELB, we create a new ELB using the same (or similar custom) subnet and security group as the public nodes with internet access. Add listeners to map public ports on the CNAME to serviced ports on the public proxy nodes.
\ No newline at end of file
+If we do not want to have multiple "A" records, we can use a load balancer, such as EC2's ELB, for the service CNAME address. In doing such, we will create a new a load balancer instance for the ConductR services as this allows the load balancer single health check to use ConductR-HAProxy's health endpoint, not marathon-lb's.  In the example of AWS ELB, we create a new ELB using the same (or similar custom) subnet and security group as the public nodes with internet access. Add listeners to map public ports on the CNAME to serviced ports on the public proxy nodes.
```

Comment:
But this is the DC/OS CloudFormation section which is on AWS EC2, so is it ok to leave it as is?